test/serving.bash

```diff
@@ -181,20 +183,20 @@ function run_knative_serving_rolling_upgrade_tests {
 function run_knative_serving_operator_tests {
   logger.info 'Running Serving operator tests'
   (
-  local exitstatus=0
+  local exitstatus test_namespace_saved
+  exitstatus=0
   checkout_knative_serving_operator
 
-  # shellcheck disable=SC2030
-  export TEST_NAMESPACE="knative-serving"
+  test_namespace_saved="${TEST_NAMESPACE}"
```

Comment:
Same question as above. Does this matter given we're in a subshell?

Reply:
Same answer as https://github.com/openshift-knative/serverless-operator/pull/267#discussion_r411209459